widgetssdk/src/main/java/com/glia/widgets/helper/DialogHolderActivity.kt

```diff
@@ -26,6 +26,11 @@ internal class DialogHolderActivity : FadeTransitionActivity() {
         setContentView(DialogHolderView(this))
     }
 
+    override fun onPause() {
+        super.onPause()
+        finish()
```

Comment:
This seems to me a bit dangerous.
Calling one lifecycle method in another one could lead to unpredictable results.
Lets try to fix the cause of the problem.

Reply:
I guess I found a better solution.

Reply:
https://github.com/salemove/android-sdk-widgets/pull/1065/files#diff-e12c01bfa33793e79d7ece617032b133f97f172f3502dd1e2726770ea52135ebR37

Reply:
Looks good!